app/Config/config.dist.json

```diff
@@ -97,6 +97,22 @@
                 "hub": "127.0.0.1",
                 "path": "/api/username/lights/1/state"
             }
+        },
+        "WebHook": {
+            "globalConfig": {
+                "failure": {
+                    "hostName": "maker.ifttt.com",
+                    "path": "/trigger/failure/with/key/your_ifttt_token"
```

Comment:
Maybe I'm being difficult, but I think it would be cleaner/easier if you can just provide 1 `url` here. What do you think?

Reply:
@DennisSnijder poke ;)

Reply:
@eXistenZNL oh wow, that's been a while. I'll have a look later on 😄

Reply:
:innocent: